kiva/fonttools/_scan_parse.py

```diff
@@ -194,34 +196,40 @@ def _afm_font_property(fontpath, font):
 
     #  All AFM fonts are apparently scalable.
     size = "scalable"
-    return FontEntry(fontpath, name, style, variant, weight, stretch, size)
+    return FontEntry(fontpath, family, style, variant, weight, stretch, size)
 
 
-def _ttf_font_property(fpath, font):
+def _ttf_font_property(fpath, font, face_index=0):
     """ A function for populating the :class:`FontEntry` by extracting
     information from the TrueType font file.
 
     *font* is a :class:`TTFont` instance.
     """
     props = get_ttf_prop_dict(font)
-    name = props.get("name")
-    if name is None:
-        raise KeyError("No name could be found for: {}".format(fpath))
+    family = props.get("family")
+    if family is None:
+        raise KeyError("No family could be found for: {}".format(fpath))
+
+    # Some properties
+    full_name = props.get("full_name", "").lower()
+    style_prop = props.get("style", "").lower()
+    if style_prop == "":
+        # For backwards compatibility with previous parsing behavior
+        style_prop = full_name
 
     #  Styles are: italic, oblique, and normal (default)
-    sfnt4 = props.get("sfnt4", "").lower()
-    if sfnt4.find("oblique") >= 0:
+    if style_prop.find("oblique") >= 0:
         style = "oblique"
-    elif sfnt4.find("italic") >= 0:
+    elif style_prop.find("italic") >= 0:
         style = "italic"
     else:
         style = "normal"
 
     #  Variants are: small-caps and normal (default)
-    # NOTE: Not sure how many fonts actually have these strings in their name
+    # NOTE: Not sure how many fonts actually have these strings in their family
     variant = "normal"
-    for value in ("capitals", "small-caps"):
-        if value in name.lower():
+    for value in ("capitals", "small-caps", "smallcaps"):
```

Comment:
I made this addition because there's a font named "Bodoni 72 Smallcaps Book" on macOS.